test/components/structures/TimelinePanel-test.tsx

```diff
@@ -570,10 +571,11 @@ describe("TimelinePanel", () => {
 
         const { container } = render(
             <MatrixClientContext.Provider value={client}>
-                <TimelinePanel {...props} />
+                <TimelinePanel timelineSet={timelineSet} manageReadReceipts={true} sendReadReceiptOnLoad={true} />
             </MatrixClientContext.Provider>,
         );
 
-        expect(container.querySelectorAll(".mx_EventTile")).toHaveLength(5);
+        await waitFor(() => expect(screen.queryByRole("progressbar")).toBeNull());
+        await waitFor(() => expect(container.querySelector(".mx_RoomView_MessageList")).not.toBeEmptyDOMElement());
```

Comment:
this feels a bit brittle: checking that there are `EventTile`s felt much more robust.